src/languages/en.ts

```diff
@@ -952,17 +951,10 @@ const translations = {
         yourCompanyWebsiteNote: "If you don't have a website, you can provide your company's LinkedIn or social media profile instead.",
         invalidDomainError: 'You have entered an invalid domain. To continue, please enter a valid domain.',
         publicDomainError: 'You have entered a public domain. To continue, please enter a private domain.',
-        expenseCount: ({scanningReceipts = 0, pendingReceipts = 0}: RequestCountParams) => {
```

Comment:
Can we make sure that this doesn't affect the existing report previews? This feels more like a global change... whereas we just want to change this up for the new report preview style we have, which is behind a beta.

Reply:
cc @mountiny for thoughts on that